lib/src/test/java/io/ably/lib/test/realtime/RealtimeChannelFailedTest.java

```diff
@@ -1,8 +1,60 @@
 package io.ably.lib.test.realtime;
 
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import io.ably.lib.realtime.AblyRealtime;
+import io.ably.lib.realtime.Channel;
+import io.ably.lib.realtime.ChannelState;
+import io.ably.lib.realtime.ChannelStateListener;
+import io.ably.lib.realtime.ConnectionState;
+import io.ably.lib.realtime.ConnectionStateListener;
+import io.ably.lib.test.common.Helpers;
+import io.ably.lib.test.common.Setup;
+import io.ably.lib.types.AblyException;
+import io.ably.lib.types.ClientOptions;
+import io.ably.lib.types.ErrorInfo;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
 /**
  * Created by kostiantyn on 8/9/2016.
  */
 
 public class RealtimeChannelFailedTest {
+    @BeforeClass
+    public static void setUpBeforeClass() throws Exception {
+        Setup.getTestVars();
+    }
+
+    @AfterClass
+    public static void tearDownAfterClass() throws Exception {
+        Setup.clearTestVars();
+    }
+
+    @Test
+    public void channel_filed_state_test(){
```

Comment:
What does this name mean?  I expect it's the typo that is confusing things, but it's still not describing what about the failed state you are testing.